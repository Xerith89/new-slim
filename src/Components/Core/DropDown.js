import React from 'react'

export default function DropDown() {
    return (
        <div data-testid="menu">
            <ul className="list-group dropdown-menu dropdown-menu-right">
                <li className="dropdown-item"><a href="/settings">Settings</a></li>
                <div className="dropdown-divider"></div>
                <li className="dropdown-item"><a href="/logout">Logout</a></li>
            </ul>
        </div>
    )
}