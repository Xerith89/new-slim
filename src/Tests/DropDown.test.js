import React from 'react';
import DropDown from '../Components/Core/DropDown';
import {render} from '@testing-library/react';

test('DropDown component renders to the DOM', () => {
    const {getByTestId} = render(<DropDown />);
    expect(getByTestId('menu')).toMatchSnapshot();
});
