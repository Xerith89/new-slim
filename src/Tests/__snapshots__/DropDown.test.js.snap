// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropDown component renders to the DOM 1`] = `
<div
  data-testid="menu"
>
  <ul
    class="list-group dropdown-menu dropdown-menu-right"
  >
    <li
      class="dropdown-item"
    >
      <a
        href="/settings"
      >
        Settings
      </a>
    </li>
    <div
      class="dropdown-divider"
    />
    <li
      class="dropdown-item"
    >
      <a
        href="/logout"
      >
        Logout
      </a>
    </li>
  </ul>
</div>
`;
